tests/foreman/cli/test_gpgkey.py

```diff
@@ -125,10 +90,11 @@ def test_redmine_4272(self):
         @BZ: Redmine#4272
 
         """
-
         # GPG Key data
-        data = {'name': gen_string("alpha", 10)}
-        data['organization-id'] = self.org['id']
+        data = {
+            'name': gen_string("alpha", 10),
```

Comment:
Not touched this to maintain focus on the PR purpose, but I think worth changing this and will handle.